tests/model_hub_tests/performance_tests/test_tf_hub_perfomance_model.py

```diff
@@ -0,0 +1,87 @@
+# Copyright (C) 2023 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+
+import os
+import shutil
+
+import gc
+import tarfile
+import tempfile
+import pytest
+import tensorflow as tf
+import openvino as ov
+import tensorflow_hub as hub
+# noinspection PyUnresolvedReferences
+import tensorflow_text  # do not delete, needed for text models
```

Comment:
btw, `tensorflow_text` is not needed since you do not load a model into memory and convert it from disk